test/non_existent_command.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+load test_helpers
+
+setup() {
+  setup_asdf_dir
+  install_dummy_plugin
+  install_dummy_legacy_plugin
+  run asdf install dummy 1.0
+  run asdf install dummy 1.1
+
+  PROJECT_DIR="$HOME/project"
+  mkdir -p "$PROJECT_DIR"
+}
+
+teardown() {
+  clean_asdf_dir
```

Comment:
Is this needed? You don't actually install anything so I'm not sure this clean function needs to run.

Reply:
Not sure.

I see it removes some environment variables.

I removed the `teardown()` function.